lib/ceramicFunctions.js

```diff
@@ -0,0 +1,174 @@
+import { CeramicClient } from '@ceramicnetwork/http-client'
```

Comment:
Did you copy this file?
If yes, add a comment with the link on top.

Reply:
Nope. It's just a module and it's on the package.json

Reply:
Then you did great work with writing explanatory comments 👌🏾